src/wallets/hardware/LedgerWallet.ts

```diff
@@ -96,9 +97,7 @@ class LedgerWallet implements HardwareWallet {
 
   // TODO: Migrate to storage service (with JSON support).
   private getAuthData = (): AuthData | null => {
-    const authData = localStorage.getItem(
-      LOCAL_STORAGE_LEDGER_WALLET_AUTH_DATA
-    );
+    const authData = storage.getItem(LOCAL_STORAGE_LEDGER_WALLET_AUTH_DATA);
```

Comment:
It would be really good if we could embed JSON support (hence why it's used for all the values for easy migration) into the storage service. It'll resolve the TODOs and remove class methods like this